adafruit_ads1x15/ads1x15.py

```diff
@@ -222,6 +251,22 @@ def _write_register(self, reg: int, value: int):
         with self.i2c_device as i2c:
             i2c.write(self.buf)
 
+    def write_comparator_low_threshold(self, value: int):
+        """Write 16 bit value to Comparator Low Threshold register."""
+        self.buf[0] = _ADS1X15_POINTER_LO_THRES
+        self.buf[1] = (value >> 8) & 0xFF
+        self.buf[2] = value & 0xFF
+        with self.i2c_device as i2c:
+            i2c.write(self.buf)
+
+    def write_comparator_high_threshold(self, value: int):
+        """Write 16 bit value to Comparator High Threshold register."""
+        self.buf[0] = _ADS1X15_POINTER_HI_THRES
+        self.buf[1] = (value >> 8) & 0xFF
+        self.buf[2] = value & 0xFF
+        with self.i2c_device as i2c:
+            i2c.write(self.buf)
+
```

Comment:
I'd make these two properties since they are comparator state.

Reply:
New comparator_low_thres and comparator_high_thres properties with setter/getter functions added. These 2 write functions consolidated into a single "write comparator thresholds" function.